src/test/ui/nll/match-cfg-fake-edges.stderr

```diff
@@ -1,8 +1,11 @@
-error[E0381]: use of possibly-uninitialized variable: `x`
+error[E0381]: used binding `x` isn't initialized in all conditions
   --> $DIR/match-cfg-fake-edges.rs:21:13
    |
+LL |     let x;
+   |         - binding declared here but left uninitialized
+...
 LL |             x;
-   |             ^ use of possibly-uninitialized `x`
+   |             ^ `x` used here but it isn't initialized in all conditions
```

Comment:
I think this error is missing the assigned-here hints, because it didn't give any branch-where-missing hints.

Or it needs new text? "Assigned in unrelated match condition"

Reply:
This is interesting, it should indeed be pointing at `x = 2` in the other match arm guard.

Reply:
Figured it out. Changed it to be:

```

error[E0381]: used binding `x` isn't initialized
  --> src/test/ui/nll/match-cfg-fake-edges.rs:21:13
   |
15 |     let x;
   |         - binding declared here but left uninitialized
...
21 |             x; //~ ERROR E0381
   |             ^ `x` used here but it isn't initialized
```

Don't know if I should point at the `x = 2` in the other match arm guard 🤔

Reply:
Changed it to be

```
error[E0381]: used binding `x` isn't initialized
  --> src/test/ui/nll/match-cfg-fake-edges.rs:21:13
   |
15 |     let x;
   |         - binding declared here but left uninitialized
...
19 |         _ if { x = 2; true } => 1,
   |                ----- binding initialized here in some conditions
20 |         _ if {
21 |             x; //~ ERROR E0381
   |             ^ `x` used here but it isn't initialized
```